conduct.md

```diff
@@ -29,26 +29,35 @@ Allegheny's honor code for all submitted work.
     * Allow peers to voice their opinions and questions.
     * Read, understand, and follow all parts of this code of Conduct.
     * Follow the honor code on all submitted work.
-    * :heavy_exclamation_mark: TODO: add any additional items in the A category.
+    * Respect, listen, and consider all ideas and contributions made by teammates
```

Comment:
This point is very similar to `Show respect for all peers as learners and team member`. You should consider changing them to avoid repetition